install_config/web_console_customization.adoc

```diff
@@ -991,16 +972,54 @@ assetConfig:
 +
 ----
 ifdef::openshift-origin[]
-# systemctl restart origin-master
+# systemctl restart origin-master-api origin-master-controllers
 endif::[]
 ifdef::openshift-enterprise[]
-# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 endif::[]
 ----
 
 xref:../install_config/router/default_haproxy_router.adoc#using-wildcard-routes[Learn
 how to configure HAProxy routers to allow wildcard routes].
 
+[[web-console-enable-tech-preview-feature]]
+== Enabling Features in Technology Preview
```

Comment:
This bit as about the web console specifically, correct? Might be better to call that out: 

Enabling Technology Previews features to show in the web console

Reply:
Follow-up in https://github.com/openshift/openshift-docs/pull/7528